docs/reference/commandline/dockerd.md

```diff
@@ -828,6 +829,24 @@ To set the DNS search domain for all Docker containers, use:
 $ sudo dockerd --dns-search example.com
 ```
 
+#### Air-gapped registries
+
+By default, Docker does not push foreign layers to registries. To
```

Comment:
@cpuguy83: Renamed the option to `--allow-nondistributable-artifacts` and took a stab at the requested explanation in 598f1c2e12dcd6937a05ac26bd8690f3f031ca86.